### ISSUE TITLE
server send events

### ISSUE THREAD

**Issue by piranna** [mindflayer/python-mocket#243]
I would like to test Server-Send Events, not sure if it should be added to the HTTP animal or an extension of it. Basically, it's a regular HTTP connection that it's left open and server keeps sending data to it at the same time is being generated, sort of streaming. Any posibility to be added as an animal?

**Comment by mindflayer** [comment]
I don't believe that having a socket that is left open for a while would be an issue, but I may be wrong. I haven't spent time on trying to make websockets work in ages.

Do you have a non-working example using Mocket? I mean: let's start from what works today and whatnot.

**Comment by piranna** [comment]
No, i don't have an example, didn't tried It yet.

**Comment by mindflayer** [comment]
Hi @piranna, are you considering helping with this, or it was just a request?
I am asking because I believe I won't be able to help with that if it's everything on me.

**Comment by piranna** [comment]
I forgot on that. I could help, but we are out of free time at work. I could help solving doubts by the moment.
